db/schema.rb

```diff
@@ -11,17 +11,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20190423235125) do
+ActiveRecord::Schema.define(version: 20190426213534) do
 
   create_table "recipes", force: :cascade do |t|
-    t.datetime "created_at",   null: false
-    t.datetime "updated_at",   null: false
-    t.integer  "day"
-    t.integer  "slot"
+    t.datetime "created_at", null: false
```

Comment:
will these be overwritten automatically when we apply the migrations? So we don't have to worry about merge conflicts

Reply:
Yes, from my understanding, running rake db:migrate applies new migrations, which in turn rewrites the schema.rb file